### PR TITLE
Resolved npm run watch missing dist file error by adding the nimbus.json schema file to the parseManifest function

### PR DESCRIPTION
**Pull request by thessakranendonk**

### CODE REVIEW

**Comment by thessakranendonk** [comment]
Please let me know if this is the correct solution :)

**Review by brennie — CHANGES_REQUESTED** [review]
Ideally commit message summaries should be in the imperative mood, i.e., tell me what the commit does, not what it did.

e.g., I would rewrite this commit message as:

"Always copy API schemas in manifest build task"

File: bin/build-manifest.mjs [hunk 1]
```diff
@@ -30,6 +30,11 @@ function parseManifest({ absolute = false } = {}) {
   files.push(...(manifest.background?.scripts ?? []));
 
   for (const api of Object.values(manifest.experiment_apis ?? {})) {
+    if (api.schema) {
+      // nimbus.json schema
+      files.push(api.schema);
+    }
```
[the remark this answers is not in the document]
Comment (thessakranendonk) [reply bin/build-manifest.mjs]: Perfect thank you! I  have updated the code.

**Review by brennie — CHANGES_REQUESTED** [review]

File: bin/build-manifest.mjs [hunk 2]
```diff
@@ -30,6 +30,9 @@ function parseManifest({ absolute = false } = {}) {
   files.push(...(manifest.background?.scripts ?? []));
 
   for (const api of Object.values(manifest.experiment_apis ?? {})) {
+    // nimbus.json schema
```
[the remark this answers is not in the document]
Comment (thessakranendonk) [reply bin/build-manifest.mjs]: Removed it!

**Review by brennie — APPROVED** [review]

**Comment by brennie** [comment]
Can you squash your commits together into a single commit?

**Comment by thessakranendonk** [comment]
Sure! Did not know that was possible haha! All done. How would you like the merging process to go? Does the reviewer merge or the branch owner?

**Comment by brennie** [comment]
You should have permission to click the "Rebase and merge" button, so I'll let you do the honours.

**Comment by brennie** [comment]
Fixes #2

**Pull request merged**